packages/ketcher-react/src/script/render/restruct/resimpleobject.js

```diff
@@ -366,9 +343,23 @@ ReSimpleObject.prototype.makeSelectionPlate = function (
     return scale.obj2scaled(p, restruct.render.options) || new Vec2()
   })
 
-  const path = generatePath(this.item.mode, paper, pos)
-
-  return path.attr(styles.highlightStyleSimpleObject)
+  const refPoints = this.getReferencePoints()
+  const s = restruct.render.options.scale
+  var st = restruct.render.paper.set()
```

Comment:
please add more meaningfull names insted of `s` and `st`

Reply:
fixed